### PR TITLE
Fix workflows

### PR DESCRIPTION
**Pull request by takurinton**
https://github.com/kuma-ui/kuma-ui/pull/265#issuecomment-1666664520

- `.github/workflows/main.yml` should only be run when `packages/**` changes
- Exclude the `experimental` branch as it is not used

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,11 +3,12 @@ on:
...
     branches:
       - main
-      - experimental
   pull_request:
     branches:
       - main
-      - experimental
+    paths:
+      - "packages/**"
+      - ".github/workflows/main.yml"
```
Comment (takurinton) [inline .github/workflows/main.yml:11]: In addition to packages, CI should also run if this file itself changes.

**Comment by poteboy** [comment]
@takurinton 
Thanks for the PR!
One question: if GitHub Actions only run when changes are made to the packages/** directory, what happens when we want to modify the website? It seems like changes to the website should also trigger a build? Correct me if I'm wrong xD

**Comment by takurinton** [comment]
@poteboy 

I'm thinking that the main.yml CI doesn't need to be running since the vercel CI does the website changes.

**Comment by poteboy** [comment]
Oh that's right lol

**Comment by takurinton** [comment]
Also, since this part remains, it will be deployed to the production!!

https://github.com/kuma-ui/kuma-ui/pull/267/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R2-R5

**Comment by takurinton** [comment]
Ready for review

**Review by poteboy — APPROVED** [review]
🐜1️⃣0️⃣

**Pull request merged**